source/js/IngredientsInfo.js

```diff
@@ -0,0 +1,309 @@
+    let shadow = this.attachShadow({ mode: "open" });
+  }
+
+  set data(data) {
+    const style = `
+        .ingredients-info {
+          position: absolute;
+          width: 20%;
+          height: 66%;
+          top: 20%;
+          left: 2%;
+          background: #FFF6EC;
+      }
+      
+      .ingredients-label {
+          position: absolute;
+          left: 22%;
+          top: 9%;
+          font-family: DM Sans;
+          font-style: normal;
+          font-weight: normal;
+          font-size: 32px;
+          line-height: 62px;
+      }
+
+      .line {
+        margin-top: 36%;
+        width: 90%;
+        text-align: center;
+        color: black;
+      }
+
+      .form > button {
+          position: absolute;
+          font-family: DM Sans;
+          font-style: normal;
+          font-weight: normal;
+          font-size: 18px;
+          height: 8%;
+          background: #FFB673;
+      }
+      
+      .cartButton {
+        left: 6%;
+        top: 20%;
+      }
+
+      .addButton {
+        left: 45%;
+        top: 30%;
+        height: 8%;
+      }
+
+      .subtractButton {
+        left: 25%;
+        top: 30%;
+        height: 5%;
+      }
+
+      .quantity {
+          position:absolute;
+          left:36%;
+          top: 25%;
+          font-family: DM Sans;
+          font-style: normal;
+          font-weight: normal;
+          font-size: 32px;
+          line-height: 30px;
+      }
+      
+      #serving_size {
+          position:absolute;
+          left:12%;
+          top: 38%;
+          width:98%;
+          font-family: DM Sans;
+          font-style: normal;
+          font-weight: normal;
+          font-size: 12px;
+          line-height: 20px;
+      }
+      
+      .ingredients-list {
+          position:absolute;
+          left: 20%;
+          top: 48%;
+          width: 60%;
+          font-family: DM Sans;
+          font-size: 12px;
+          -ms-transform: scale(1.5); 
+          -moz-transform: scale(1.5); 
+          -webkit-transform: scale(1.5); 
+          -o-transform: scale(1.5); 
+          transform: scale(1.5);
+          padding: 10px;
+      }
+      `;
+
+    const styleElem = document.createElement("style");
+    styleElem.innerHTML = style;
+
+    let ingredientList = searchForKey(data, "recipeIngredient");
+
+    const info = document.createElement("div");
+    info.classList.add("ingredients-info");
+
+    let ingredients = document.createElement("p");
+    ingredients.classList.add("ingredients-label");
+    ingredients.textContent = "Ingredients";
+    info.appendChild(ingredients);
+
+    let line = document.createElement("hr");
+    line.classList.add("line");
+    info.appendChild(line);
+
+    let form = document.createElement("div");
+    form.classList.add("form");
+
+    let addToCart = document.createElement("button");
+    addToCart.classList.add("cartButton");
+    addToCart.textContent = "Add Ingredients To Cart";
+    form.appendChild(addToCart);
```

Comment:
Codacy has a fix for the issue: [Prefer `Node#append()` over `Node#appendChild()`.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8447291)
```suggestion
    form.append(addToCart);
```